src/include/migraphx/half.hpp

```diff
@@ -25,7 +25,7 @@
 #ifndef MIGRAPHX_GUARD_RTGLIB_HALF_HPP
 #define MIGRAPHX_GUARD_RTGLIB_HALF_HPP
 
-#include <half/half.hpp>
+#include <half.hpp>
```

Comment:
This should be `half/half.hpp`. Thats a requirement for rocm packages.

Reply:
Noted